W-16993325-pce4-documentation-kt (#344)
mulesoft/docs-private-cloud: modules/ROOT/pages/config-workflow.adoc

= Configuring Anypoint Platform PCE
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:page-aliases: install-add-tasks.adoc

> After you complete Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) installation, review the following tasks, so you can successfully deploy and manage applications, register APIs, and use API Exchange on an Anypoint Platform PCE environment running on your own local network.

Reviewer: ```suggestion
After you complete the Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) installation, review the following tasks to deploy and manage applications, register APIs, and use API Exchange on an Anypoint Platform PCE environment running on your local network.
```
